app/controllers/admin/rebate_forms_controller.rb

```diff
@@ -62,7 +64,7 @@ def set_rebate_form
   end
 
   def rebate_form_params
-    params.require(:rebate_form).permit(attachments: [])
+    params.fetch(:rebate_form, {}).permit(:last_edited, {attachments: []}, :valuation_id, :address, :income, :dependants, :full_name, :phone_number, :rates_bill, :lived_here_before_july_2018, :income_range, :email, :has_home_business, :email_phone_can_be_used)
```

Comment:
i've not used `.fetch` before. why do we need it an not require?

Reply:
how about 
```
.permit(fields: [:income, :dependants, :full_name, :phone_number, .....])
```

So that the fields stay nested in their own hash.

Reply:
I really don't think rates_bill should be editable. It also shouldn't be in the `fields`.